data/json/items/containers.json

```diff
@@ -112,7 +112,10 @@
         "to_hit":-1,
         "material":"steel",
         "contains":1,
-        "rigid": true
+        "rigid": true,
+        "watertight": true,
+        "seals": true,
+        "qualities": [["CONTAIN", 1]]
```

Comment:
well, please compare this with plastic bowl(line 568) and clay bowl(line 761). They both have these flags.